src/pages/invoice/create.html

```diff
@@ -0,0 +1,179 @@
+
+      <div class="section section--sb">
+        <div class="billing-details">
+          <div>
+            <span class="billing-details__heading">Bill to</span>
+            <div class="billing-details__content">Buck Miller 8889913444 buck@miller.com</div>
+          </div>
+          <button class="btn btn--text">Change</button>
+        </div>
+        <div class="input-details">
+          <label for="issue_date"
+            >Issued At
+            <div class="input_container">
+              <input type="text" name="issue_date" class="input" value="06 Aug 2020" />
+              <i class="material-icons input_icon">calendar_today</i>
+            </div>
+          </label>
+          <label for="due"
+            >Due Date
+            <div class="input_container">
+              <input type="text" name="due" class="input" value="13 Aug 2020" />
+              <i class="material-icons input_icon">calendar_today</i>
+            </div>
+          </label>
+          <label for="inv_no"
+            >Invoice Number
+            <div class="input_container">
+              <input type="text" name="inv_no" class="input" value="INV - 000011" />
+              <i class="material-icons input_icon">tag</i>
+            </div>
+          </label>
+          <label for="ref_no"
+            >Ref Number
+            <div class="input_container">
+              <input type="text" name="ref_no" class="input" value="" />
+              <i class="material-icons input_icon">tag</i>
+            </div>
+          </label>
+        </div>
+      </div>
+
+      <div class="section">
+        <div class="scrollable">
+          <table>
+            <thead>
+              <tr>
+                <th>Items</th>
+                <th>Quantity</th>
+                <th>Price</th>
+                <th>Amount</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>Model S</td>
+                <td>
+                  <input type="number" value="2" />
+                </td>
+                <td>&#8377;350</td>
+                <td>
+                  &#8377;700
+                  <i class="material-icons row-delete">delete</i>
+                </td>
+              </tr>
+              <tr>
+                <td>Model 3</td>
+                <td>
+                  <input type="number" value="1" />
+                </td>
+                <td>&#8377;800</td>
+                <td>
+                  &#8377;800
+                  <i class="material-icons row-delete" role="button">delete</i>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+        <button class="btn btn--text add-btn">
+          <i class="material-icons btn__icon--large btn__icon--left" role="button">
+            shopping_basket
+          </i>
+          Add an Item
+        </button>
+        <div class="divider divider--full"></div>
+      </div>
+
+      <div class="section section--sb">
```

Comment:
Try to look at using more semantic elements that just div/span.

Would suggest to look at tags like `section`,`article`, `aside` etc..